Gemfile

```
@@ -115,6 +115,11 @@ gem "dalli"
 # Load canonical-rails to generate canonical URLs
 gem "canonical-rails"
 
+# Use to generate telemetry
+gem "opentelemetry-exporter-otlp"
+gem "opentelemetry-instrumentation-all"
+gem "opentelemetry-sdk"
```

These are `require`d by default. If we are going to `require` them only when `OTEL_TRACES_EXPORTER` is defined, then we should add `require: false` to each:

```suggestion
gem "opentelemetry-exporter-otlp", require: false
gem "opentelemetry-instrumentation-all", require: false
gem "opentelemetry-sdk", require: false
```

You can verify this by running: `bin/rails runner OpenTelemetry::SDK`. With the code in this PR, it runs without a complaint. If you add the `require: false` you'll see it end with `uninitialized constant OpenTelemetry`.

---

Thanks. I've made those changes now.